setup.py

```diff
@@ -43,7 +43,8 @@
     "Mathics-Scanner >= 1.3.0.dev0",
     # Pillow 9.1.0 supports BigTIFF with big-endian byte order.
     # ExampleData image hedy.tif is in this format.
-    "pillow >= 9.1.0",
+    # Pillow 9.2 handles sunflowers.jpg
+    "pillow >= 9.2.0",
```

Comment:
I am finding problems in installing pillow 9.4.0  on anaconda / pyston 2.3.4 (3.8.12)